Basic-Car-Maintenance/Shared/Settings/Views/AuthenticationView.swift

```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct AuthenticationView: View {
     @Environment(\.colorScheme) var colorScheme
     
-    @ObservedObject var viewModel: AuthenticationViewModel
+    var viewModel: AuthenticationViewModel
```

Comment:
I think this needs to be `@Bindable`

Reply:
here we do not use `$viewmodel.xxxxx` so according to the docs, this would not have to be `@Bindable`